packages/runtime/src/state/modules/api-resources.ts

```diff
@@ -1,14 +1,35 @@
-import { APIResourceType, APIResourceLocale, type APIResource } from '../../api'
+import {
+  APIResourceType,
+  APIResourceLocale,
+  LocalizableAPIResourceType,
+  type APIResource,
+} from '../../api'
 
 import deepEqual from '../../utils/deepEqual'
 import { Branded } from '../../utils/branded'
 
-import { Action, ActionTypes } from '../actions'
+import { type Action, ActionTypes } from '../actions'
 
 type CompositeResourceId = Branded<string, 'CompositeResourceId'>
 
-function compositeId(resourceId: string, locale?: string | null): CompositeResourceId {
-  return (locale != null ? `${resourceId}@${locale}` : resourceId) as CompositeResourceId
+function isValidAPIResourceType(resourceType: string): resourceType is APIResourceType {
+  return resourceType in APIResourceType
+}
+
+function isLocalizableAPIResourceType(resourceType: APIResourceType): boolean {
+  return resourceType in LocalizableAPIResourceType
+}
```

Comment:
Should we make this predicate a type guard like we did for `isValidAPIResourceType`?